versioned_docs/version-2.x/advanced/analytics/getting-started.mdx

```diff
@@ -185,3 +228,47 @@ module.exports = {
 ```
 
 :::
+
+### Granular Consent Updates
+
+When using custom consent options defined in your `cookiesServices.js` file, you
+can implement granular consent updates by adding an `updateConsent` method to
+your analytics plugin. This method allows you to handle consent changes for
+specific tracking features.
+
+The `updateConsent` method is already included in the built-in
+`google-analytics` and `google-tag-manager` plugins, but you'll need to
+implement it yourself for custom plugins.
+
+Here's how to add consent updates to your plugin:
+
+```ts title="app/config/analytics.js"
```

Comment:
Same comment as above, I think the path is `src/` in v2

```suggestion
```ts title="src/config/analytics.js"
```